site/content/docs/howto/debugging.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Debugging Pinniped 
+description: Debugging `pinniped` configuration errors.
+description: 
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: Debugging
+    weight: 900
+    parent: howtos
+---
+
+## Debugging on the Client 
+
+The `PINNIPED_DEBUG=true` environment variable can be set to enable additional CLI logging.
+
+## Debugging on the Server
+
+To adjust the log level of either the Pinniped Supervisor or the Pinniped Concierge the log level value must be updated 
+in the appropriate configmap associated with each deployment. 
+
+The `log level` options are as follows: 
+- `info` ("nice to know" information) 
+- `debug` (developer information) 
+- `trace` (timing information)
+- `all` (kitchen sink)
+
+Do not use `trace` or `all` on production systems, as credentials may get logged. When this value is left unset, 
+only warnings and errors are printed. There is no way to suppress warning and error logs.
+
+Choose an update method that corresponds with the original installation method you chose for your cluster. Consult
+[Install Supervisor]({{< ref "../howto/install-supervisor" >}}) or [Install Concierge]({{< ref "../howto/install-concierge" >}})
+for more information.
+
+### Using kapp
+
+To adjust the log level of the Pinniped Supervisor or Concierge using `kapp`, edit the `log_level` in the `values.yaml` files 
```

Comment:
I think you meant to say using `ytt`, not using `kapp`.

And you don't edit the values.yaml file. See the other docs for instructions.

Reply:
Ah, yeah I see we tell users to create a NEW `values.yaml` file and add just the config they desire to change:

```yaml
# ./my-supervisor-values.yaml

#@data/values
---
image_tag: v0.29.0
log_level: debug
replicas: 1
```
And then something `ytt --file /path/to/pinniped/deploy/supervisor --file ./my-supervisor-values.yaml`